.github/workflows/ci.yml

```diff
@@ -369,7 +369,7 @@ jobs:
         gcc --version
         .\make.bat -gcc
     - name: Test new v.c
-      run: .\v.exe -o v.c cmd/v && gcc -Werror -municode -w v.c
```

Comment:
I think `-Werror` should stay. 
It ensures that the corresponding compiler can compile V itself without new warnings. 
There are equivalents for the other targets too.

Reply:
I agree :slightly_smiling_face: .